src/style.scss

```diff
@@ -0,0 +1,145 @@
+$bodyFont: roboto, helvetica, sans-serif;
+$main-color: #060b35;
+$secondary-color: #616070;
+
+body {
+  margin: 0;
+  font-family: $bodyFont;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 406px;
+  flex-shrink: 0;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding-top: 32px;
+
+  &__icon {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    // margin: 0;//
+  }
+
+  &__container {
+    margin-top: 40px;
+    padding: 0 16px 16px;
+  }
+
+  &__title {
+    margin: 0;
+    color: $main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-top: 4px;
+    margin-bottom: 16px;
+    color: $secondary-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+  }
+
+  &__numberOfStars {
```

Comment:
Consider naming classes accordingly to BEM conventions instead of camel case notation